ament_cmake_pytest/cmake/ament_has_pytest_cov.cmake

```diff
@@ -0,0 +1,74 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Check if the Python module `pytest-cov` was found and get its version if it is.
+#
+# :param var: the output variable name
+# :type var: string
+# :param version_var: the variable name for the version of pytest-cov found, only
+#   set if it was found
+# :type version_var: string
+# :param QUIET: suppress the CMake warning if pytest-cov is not found, if not set
+#   and pytest-cov was not found a CMake warning is printed
+# :type QUIET: option
```

Comment:
Since the `QUIET` is currently always used maybe just remove the argument if there is no current use case for it to be called without?

Reply:
Sure; fixed in 64cca30d4b7009ec757d706cef65d68c05b2d64f.

Same could be said for `ament_has_pytest()`, though https://github.com/ament/ament_cmake/blob/c66c6a0607f878788fd5ac8fd17151222c4238c0/ament_cmake_pytest/cmake/ament_add_pytest_test.cmake#L75.

Reply:
oops, another fix in eb0b5c5d4c14ec0911042d5a151078cadec37d21.